### PR TITLE
Divide function

### PR DESCRIPTION
**Pull request by LauraTrigg**

### CODE REVIEW

File: pythoncalculator/divde.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```
Comment (aishamoolla) [inline pythoncalculator/divde.py]: The argument should be y not z

**Review by aishamoolla — CHANGES_REQUESTED** [review]

File: pythoncalculator/divde.py (hunk 1 above)
Comment (LauraTrigg) [reply pythoncalculator/divde.py]: Yay thank you

Comment (LauraTrigg) [reply pythoncalculator/divde.py]: updated

**Review by aishamoolla — APPROVED** [review]
Thanks! looks good

**Pull request merged**